debug.log

```diff
@@ -0,0 +1,117 @@
+[0712/175608.106:ERROR:crash_report_database_win.cc(469)] failed to stat report
```

Comment:
`SUGGESTION`

Consider deleting this file or had to `.gitignore` if you really want to keep it.